symbol/symbol_factory.py

```diff
@@ -21,8 +60,7 @@ def get_config(network, data_shape, **kwargs):
             num_filters = [512, -1, 512, 256, 256, 256, 256]
             strides = [-1, -1, 2, 2, 2, 2, 1]
             pads = [-1, -1, 1, 1, 1, 1, 1]
-            sizes = [[.07, .1025], [.15,.2121], [.3, .3674], [.45, .5196], [.6, .6708], \
-                [.75, .8216], [.9, .9721]]
+            sizes = get_scales(min_scale=0.15, max_scale=0.9, num_layers=len(from_layers))
```

Comment:
Is this making slightly different anchor scales so the current models will perform worse than the original parameters?

Reply:
when you pick min_scale=0.2, max_scale=0.9 you get exactly the same numbers as you used. 
I now see that it is not correct for min_scale=0.15 
I guess my calculations were wrong. I'll check it out and get back to you

Reply:
Fixed, Fully compatible now..